app/views/issues/map.html.slim

```diff
@@ -0,0 +1,54 @@
+link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"
+script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"
+script src="http://www.matchingnotes.com/javascripts/leaflet-google.js"
+
+h1 Мапа звернень
+#issues-map
+
+javascript: 
+  $(document).ready(function(){
+    var map = new L.Map('issues-map', 
+              { center: new L.LatLng(50.1192478, 26.8197633), 
+                zoom: 9});
+    var googleLayer = new L.Google('ROADMAP');
+    map.addLayer(googleLayer);
+    
+    $.get({url: '/map', dataType:"json", cache:false}).done(function(issues) { 
```

Comment:
why use lambda? you can do shorter
$.get({url: '/map', dataType:"json", cache:false}).done(addMarkers);